service/interpreter/globalVersioner.go

```diff
@@ -58,6 +61,16 @@ func (p *GlobalVersioner) IsAfterVersionOfTemporal26SDK() bool {
 	return p.version >= StartingVersionTemporal26SDK
 }
 
+func (p *GlobalVersioner) IsAfterVersionOfNoIwfGlobalVersionSearchAttribute() bool {
```

Comment:
Nit: it is not being used so can be removed. Do you put it here intentionally for consistency?

Reply:
Yes, I added it for consistency

Reply:
Yeah Sounds good. I am fine to keep it